README.md

```diff
@@ -14,14 +14,26 @@ Web app for Azure IoT PCS Remote Monitoring Solution [dotnet](https://github.com
 

```

Comment:
Probably right time to start dropping PCS from the repos and readmes. We can start from here. `Azure IoT Remote Monitoring Solution`